blue-green-upgrade/validate.sh

```diff
@@ -118,10 +116,10 @@ validate_control() {
 # 0 - when all node versions == NEW_K8S_VER
 # 1 - when a node version != NEW_K8S_VER
 validate_nodes() {
-  NODES=$(kubectl get nodes -o name)
+  NODES=$(kubectl -n default get nodes -o name)
   for NODE in ${NODES}; do
     # Find the kubelet version on each node.  This will match the gke version
-    NODE_VER=$(kubectl get "${NODE}" -o json | \
+    NODE_VER=$(kubectl -n default get "${NODE}" -o json | \
```

Comment:
can you just set the current context?  Not sure why we are setting the default namespace every time

Reply:
I get what you're saying but so far the precedent has been to set in other projects to set namespace explicitly. We can look at doing it a different way in a different PR.